Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -362,9 +377,9 @@ private class NpmPathProvider : INpmPathProvider {
             }
         }
 
-#endregion
+        #endregion
 
-#region Updating module hierarchy
+        #region Updating module hierarchy
```

Comment:
nit: If you format the document, do these #region tags stay in place? I was under the impression that they don't.